.github/ISSUE_TEMPLATE/doc-issue.md

```diff
@@ -15,5 +15,5 @@ about: Let us know what would make the GPflow documentation better
 
 
 Links:
-* [latest GPflow documentation](https://gpflow.readthedocs.io/en/develop/)
-* [latest GPflow tutorial notebooks](https://gpflow.readthedocs.io/en/develop/notebooks_file.html)
+* [latest GPflow documentation](https://gpflow.github.io/GPflow/develop/)
```

Comment:
have you done a grep to check that there are no more references to readthedocs left?